mgmtsystem/azure.py

```diff
@@ -160,6 +161,26 @@ def list_vm(self):
             "./Object/Property[@Name='Name']/text()")
         return nameList
 
+    def capture_vm(self, vm_name, resource_group, container, image_name):
+        self.logger.info("Attempting to Capture Azure VM {}".format(vm_name))
```

Comment:
This much logging detail feels like DEBUG level instead of INFO, but it looks like you're in-line with the existing style. First time checking out wrapanapi, not requesting a change just commenting :+1: